availability-store/src/lib.rs

```diff
@@ -263,6 +263,11 @@ impl Store {
 	}
 
 	/// Make a validator's index and a number of validators at a relay parent available.
+	///
+	/// This information is needed before the `add_candidates_in_relay_block` is called
+	/// since that call forms the awaited frontier of chunks.
+	/// In the current implementation this function is called in the `get_or_instantiate` at
```

Comment:
I won't block the PR on this, but in general I'd prefer to avoid these kinds of long-distance expectations of how the code should be used. It's an assumption that can easily change without noticing, unlike things like "X is called before Y".